### ISSUE TITLE
Caught SIGILL in blst_cgo_init

### ISSUE THREAD

**Issue by tacyarg** [prysmaticlabs/prysm#13037]
### Describe the bug

On older cpus some have issues supporting blst.

**Adding the following flags during build allows compatibility:**
```
CGO_CFLAGS: "-O -D__BLST_PORTABLE__"
CGO_CFLAGS_ALLOW: "-O -D__BLST_PORTABLE__"
```

**REFERENCE:**
https://github.com/bnb-chain/bsc/issues/1562
https://github.com/bnb-chain/bsc/pull/1575/commits/54ea7feb2c3eee397b3534f1e636387b37d812d7

### Error

```text
Caught SIGILL in blst_cgo_init, consult <blst>/bindinds/go/README.md.
```

### Platform(s)

Linux (x86)

### What version of Prysm are you running? (Which release)

4.0.8

**Comment by nisdas** [comment]
Yes, older CPUs need those specific flags in order to build a compatible portable binary. Trying to understand, what is the specific issue ? This isn't a bug here

Also in our next release this should work for all CPUs without any specific flags:
https://github.com/prysmaticlabs/prysm/pull/12720

**Comment by tacyarg** [comment]
ah ok, then this would be a duplicate thank you for pointing that out
